kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ParquetFileWriter.java

```diff
@@ -53,6 +53,10 @@
  * {@link ParquetWriter} through {@link RecordConsumer}.
  */
 public class ParquetFileWriter {
+    public static final String TARGET_FILE_SIZE_CONF =
+            "delta.kernel.default.parquet.writer.targetMaxFileSize";
+    public static final long DEFAULT_TARGET_FILE_SIZE = 128 * 1024 * 1024; // 128MB
```

Comment:
nit: I would prefer if we had all of these confs defined in one single file, but this is okay for now